.github/workflows/ci.yml

```diff
@@ -1,114 +0,0 @@
-name: JDBC Unit Tests
-on:
-  push:
-    branches:
-      - BABEL_1_X_DEV
-  pull_request:
-    branches:
-      - BABEL_1_X_DEV
-
-jobs:
-  extension-tests:
-    name: Build and test
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: Clone, build, and run tests for Postgres engine
-        run: |
-          cd ..
-          git clone https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish.git
-          sudo apt-get update
-          sudo apt-get install uuid-dev openjdk-8-jre libicu-dev libxml2-dev openssl libssl-dev python-dev libossp-uuid-dev libpq-dev cmake pkg-config g++ build-essential bison 
-          cd postgresql_modified_for_babelfish
-          ./configure --prefix=$HOME/postgres/ --with-python PYTHON=/usr/bin/python2.7 --enable-debug CFLAGS="-ggdb" --with-libxml --with-uuid=ossp --with-icu
-          make -j 4 2>error.txt
-          make install
-          make check
```

Comment:
make check runs the Postgres tests, as far as I can tell this command doesn't exist anymore with your changes. We probably only need to run them once, so maybe you can add another workflow (postgres.yml) to run them

Reply:
Addressed